docs/custom-flows/email-password-mfa.mdx

```diff
@@ -439,6 +439,135 @@ To authenticate a user using their email and password, you need to:
 
   </InjectKeys>
   </Tab>
+  <Tab>
+    For the UI in this example, install the `expo-checkbox` package:
+    <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+      ```bash filename="terminal"
+      npm install expo-checkbox
+      ```
+      ```bash filename="terminal"
+      yarn add expo-checkbox
+      ```
+      ```bash filename="terminal"
+      pnpm add expo-checkbox
+      ```
+    </CodeBlockTabs>
+
+    Then create a component with a custom MFA sign-in form.
+
+    > You can render this component in a custom sign-in or sign-up flow, which you can find examples of in [the Expo quickstart](/docs/quickstarts/expo).
+
+    ```tsx filename="components/SignInMFAForm.tsx"
+    import React from "react";
+    import { useSignIn } from "@clerk/clerk-expo";
+    import { useRouter } from "expo-router";
+    import { Text, TextInput, TouchableOpacity, View } from "react-native";
+    import Checkbox from "expo-checkbox";
+
+    export default function SignInMFAForm() {
+      const { signIn, setActive, isLoaded } = useSignIn();
+
+      const [email, setEmail] = React.useState("");
+      const [password, setPassword] = React.useState("");
+      const [code, setCode] = React.useState("");
+      const [useBackupCode, setUseBackupCode] = React.useState(false);
+      const [displayTOTP, setDisplayTOTP] = React.useState(false);
+      const router = useRouter();
+
+      // Handle user submitting email and pass and swapping to TOTP form
+      const handleFirstStage = () => setDisplayTOTP(true);
```

Comment:
This and below have the same concerns about the logic as from the expo-mfa.mdx section.